frontend/src/app/shared/community-agreement/community-agreement.widget.html

```diff
@@ -0,0 +1,196 @@
+<mat-card class="community-agreement-dialog" #dialogContainer>
+  <mat-card-content class="content-container">
+    <p *ngIf="has_user_agreed">
+      ***You have already accepted the agreement!***
+    </p>
+    <p *ngIf="!has_user_agreed">
+      ***You have not accepted the community agreement!***
+    </p>
+    <mat-card-header>
+      <h1>
+        Undergraduate Computer Science Experience Labs (XL) Community Standards
+        Agreement
+      </h1>
+    </mat-card-header>
+    <mat-card-content>
+      <p>
+        The Undergraduate Experience Labs (XL) community serves the Computer
+        Science Undergraduate Community at UNC-Chapel Hill.
+      </p>
+      <p>
+        The XL is dedicated to providing a harassment-free and inclusive
+        experience for everyone regardless of gender identity and expression,
+        sexual orientation, disabilities, neurodiversity, physical appearance,
+        body size, ethnicity, nationality, race, religion, or other protected
+        category. These policies are inspired by and almost entirely replicate
+        Google's Developer Community Guidelines:
+        https://developers.google.com/community-guidelines
```

Comment:
IMPT: Update this and other links so they are clickable! Maybe also change it so that the link text doesn't appear and you just click the text like "Google's Developer Community Guidelines" to go to the link since the links run off the card on mobile.

Reply:
Fixed!